src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIResponse.java

```diff
@@ -108,4 +130,8 @@ public boolean isSuccess() {
   public String getResponseBody() {
     return responseBody;
   }
+  
+  public Boolean getRetryable() {
```

Comment:
why do we need 2 method - checkRetryable and getRetryable.

Reply:
The first method : `checkRetryable` is to check whether the retry is required or not. It sets the `isRetryable` variable to true when the error message : _maximum execution time exceeded_ is encountered.
The second method `getRetryable` returns the value of the class level variable: `isRetryable`. It's a getter method.